### DISCUSSION TITLE
Other efforts to map "tribes" or "centres" of the Second Renaissance

### DISCUSSION THREAD

**Discussion by catherinet1** [life-itself/community#969] [Research]
As part of our sensemaking of the Second Renaissance ecosystem, we are interested in: **Who/what are the "centres", "tribes", or communities that make up the ecosystem?**

Why? To get a feel for the landscape in terms of what key trends there are; what similarities and differences in approach there are; around what leaders are people gathering (whether those leaders are theories, ideas, practices, traditions, organisations, individuals etc.). Where does there seem to be traction? Where does there seem to be tension? 

> Why do climate tribes matter? Tribes can help people find each other more quickly by communicating values instead of agendas. ...
> 
> Secondly, tribes are a better way of forecasting what the future looks like. By understanding climate efforts as distinct, dynamic networks of talent and funding, driven by a shared set of values, we can better understand the potential of the ideas they advocate for, as well as their risks and shortcomings. ...
> 
> Finally, the most important thing about climate tribes is that they shift the conversation from passive, “true-believer” narratives towards active, action-oriented ones. ... One can be “alarmed” about climate change, as a third of American adults apparently are, and not do anything about it. But you can’t be in a climate tribe without having an opinion about solutions.
https://nadia.xyz/climate-tribes#okay-but-really---which-is-the-best-climate-tribe

# Key Questions

## How do we identify "centres" of the field/"tribes"?

3 options:
- Data-driven
- Gut instinct
- Ask people in some way
(or a combination of the above)

This research into related efforts to map "tribes" is part of our efforts to make sense of the field and identify "centres"/"tribes" (and build on existing work).

## What is a "centre"/"tribe"? 

For example:
  - A community: a group of people that gather in some way whether in person or online, and who are united by some shared values and interests
  - ...

# Examples 

Below are some examples of related work on mapping tribes/centres of what we might call the Second Renaissance.

## Memetic Tribes (Peter Limberg and Conor Barnes, September 2018)

🔗 List of tribes: https://docs.google.com/spreadsheets/d/11Ov1Y1xM-LCeYSSBYZ7yPXJah2ldgFX4oIlDtdd7-Qw/edit#gid=0
🔗 Accompanying essay: https://medium.com/s/world-wide-wtf/memetic-tribes-and-culture-war-2-0-14705c43f6bb

- 33 "Memetic Tribes" of the political landscape, e.g. Social Justice Activists; Occupy; Post-Rationalists; Establishment Right; Trumpists; Alt-Right; Incels. 
- Most wouldn't be included as tribes or centres of the Second Renaissance. Only one clearly part of Second Renaissance: Integral Theorists
- 🔥 Dimensions mapped:
  - Telos (aspiration)
  - Sacred Values
  - Master Status
  - Existential Threats
  - Combatants
  - Campfire (physical and online spaces where the tribe gathers/hangs out, e.g. Twitter, Youtube, Campuses, Streets, etc.)
  - Chieftains
  - Mental Models
  - Forebears

## Climate Tribes (Nadia Asparouhova, November 2022)

🔗 https://nadia.xyz/climate-tribes

- 7 climate tribes influencing climate work currently:
  - Energy maximalism
  - Climate urbanism
  - Climate tech
  - Eco-globalism
  - Environmentalism
  - Neopastoralism
  - Doomerism
- 3 not yet fully-fledged tribes: Climate escapists; Regenerative Finance; Bobrossians
- Dimensions:
  - Vision/goal (described in summary notes)
  - Related movements
  - Areas of interest
  - Relevant people and orgs
  - Keywords
  - Mindset (abundant--scarce)
  - Economic growth (pro--anti)
  - Techno- (optimist--pessimist)
  - Outlook on future (optimist--pessimist)
  - Scope of concern (individualist or collectivist)

> While the media still portrays climate as a simple question of beliefs, the climate field has long moved on to diversified solutions. Whether one believes in climate change is no longer the interesting question; now it’s “What do you think is the right approach?

> People who work in and around climate don’t all believe the same things. Instead, they inhabit a parallel, mirror world that looks a lot like the non-climate world. Just like in the regular world, there are factions, politics, and competing belief systems.

> Nor do climate people all agree on the right solutions to pursue. In some cases, they believe other tribes are actively harmful to their cause. The enemy, in their minds, aren’t climate deniers, as we might have seen a decade or two ago – they’re other people working in climate.

## Twelve Tribes of Transformation (Jonathan Rowson, March 2019)

🔗 https://www.whatisemerging.com/opinions/twelve-tribes-of-transformation-awakening-the-active-ingredients-of-a-new-civilisation

## Emerge Commentary on Memetic Tribes

🔗 https://www.whatisemerging.com/opinions/stoic-vibes-man

## Approaches to the Meta-Crisis (Brandon Norgaard, 2021/2022?)

🔗 https://docs.google.com/spreadsheets/d/1Stad2QYgTEWqt1sNIUSggMQ5slQ1Bnnv/edit#gid=1145361217

- 37 "Worldview, Memetic Tribe, or Project Name", e.g. Solarpunk, Radical Feminism, Revolutionary Anti-Capitalism, Dark Renaissance, Game B, Folk-Bildung 3.0, Conscious Evolution, Metamodern Spiritualist, Web3, The Regenerative Renaissance / Bio-Transformation
- Multiple of these would probably fit within Second Renaissance field
- Dimensions:
  - Approach to Addressing the Meta-Crisis
  - Thought Perspective/[vMeme](https://en.wikipedia.org/wiki/Metamodernism#Hanzi_Freinacht_and_Nordic_metamodernism)
  - Intellectual Lineage
  - Tribe of Transformation (refers to Rowson's piece)
  - Implementation
  - Proponents
  - Associated Organizations
  - Core Information Sources

## Cohere+ "Collectives" Mapping

[TO DO: Ask WP4 project team members if we can share their spreadsheet]
- Tribes:
  - Co-creative (Spiritual Unitive/Whole Systems)
  - Collaborative (Agnostic Unitive/Whole Systems)
  - Regenerative
  - Integral
  - Living Systems
  - Polycrisis
  - Existential/spiritual
  - Indigenous
  - Metamodern
  - Artivism
  - Gift Economy/Doughnut Economy/Resource-based Economy
  - Synergy Punk
  - Solar Punk
  - Commoning
  - Web 3.0
  - SDGs
  - IDGs
  - Psychedelic
  - Synarchy
  - Game B
  - Whole-World View
- Dimensions:
  - Description
  - Core values
  - Alternative Memes
  - Most Leaned Into Quadrant
  - Most Well Known Linked Collectives
  -  Well Known/Associated Thoughtleaders
  - Related Concepts

## The Liminal Web: Mapping An Emergent Subculture Of Sensemakers, Meta-Theorists & Systems Poets (Joe Lightfoot, 2021)

**NB: this came out at same time as Life Itself's  original mapping - and we talked with Joe about it**

🔗 https://www.joelightfoot.org/post/the-liminal-web-mapping-an-emergent-subculture-of-sensemakers-meta-theorists-systems-poets

## The Flourishing of All Living Things (Naryan Wong)

"This map will be a snapshot attempting to make sense of the myriad of ways humanity is attempting to save itself." https://kumu.io/Naryan/the-flourishing-of-all-living-things

Map of maps: https://kumu.io/Naryan/the-flourishing-of-all-living-things#flourishing-of-all-living-things/mapping?focus=1

## Meta-Crisis Maps & Syntheses (Sloww)

Directory of mapping & synthesis efforts in the ecosystem: https://metacrisis.org/META-CRISIS/01.+%F0%9F%93%9D+Resources/Meta-Crisis+Maps+%26+Syntheses

## Map of Sensemaking Web (GameBInsights, 2020)

https://twitter.com/GameBInsights/status/1275770634581270529/photo/1

![Screenshot 2024-03-11 at 13 30 17](https://github.com/life-itself/community/assets/97066487/5fd5d7eb-3d0c-4020-a128-5450c3ebbf36)

## Limicon 2024

https://www.limicon2024.com/

![Screenshot 2024-03-11 at 13 24 46](https://github.com/life-itself/community/assets/97066487/955f7c4e-9508-4ee0-a3c5-98caf51759d3)

## Wisdom Age narrative map (Roote)

https://www.roote.co/wisdom-age

**Comment by matthewmccarthy11** [comment]
Some really interesting groups here, esp the memetic tribes, rowsons list, and approaches to the metacrisis. Wonder if we have an understanding of how the different 'tribes' in cohere+ are organized?

**Reply by catherinet1** [reply]
thanks for your comment and question, Matthew. 

What (more specifically) do you find interesting about those examples?

I added some notes above that have come out of meetings with  @rufuspollock  and @mattosborn re How do we identify tribes/centres of the field? This research into the examples listed here is part of our sensemaking process for the Cohere+ Mapping and visualisations, so that we build well on/in conversation with existing work.

The Cohere+ Collectives Mapping I noted above refers to a document that Andrea and Kara started -- I have added a note above that I will ask them if we can share this.

**Comment by mattosborn** [comment]
Here's the full text of _The Memetic Tribes Of Culture War 2.0_. It's a great essay and well worth a read.

[The Memetic Tribes Of Culture War 2.0.pdf](https://github.com/life-itself/community/files/13741672/The.Memetic.Tribes.Of.Culture.War.2.0.pdf)

**Reply by catherinet1** [reply]
thanks @mattosborn 🙏 

p.s. @elisapaka here is the Memetic Tribes essay

**Comment by catherinet1** [comment]
Content here has been processed to ecosystem.lifeitself.org/wiki mainly into: https://ecosystem.lifeitself.org/wiki/overview-mapping-efforts

There's also some broader discussion of the idea of Tribes in https://ecosystem.lifeitself.org/wiki/tribes-of-the-second-renaissance
